common/crypto.hpp

```diff
@@ -45,7 +47,9 @@ class PrivateKey {
 	static ExpectedPrivateKey LoadFromPEM(const string &private_key_path, const string &passphrase);
 	static ExpectedPrivateKey LoadFromPEM(const string &private_key_path);
 	static ExpectedPrivateKey Generate(const unsigned int bits, const unsigned int exponent);
-	static ExpectedPrivateKey Generate(const unsigned int bits);
+	static ExpectedPrivateKey Generate(const unsigned int bits) {
```

Comment:
Nice :smiley_cat:

Reply:
I don't get your comment. You mean that I should use `int64_t` for `MENDER_DEFAULT_RSA_EXPONENT`?

Reply:
and the `const unsigned int`. But I'm not saying we should do it here, I just thought I bring it up that it's what they recommend us doing, and we're not doing it anywhere